lib/logstash/codecs/fluent.rb

```diff
@@ -51,6 +53,18 @@ def encode(event)
     @on_event.call(event, MessagePack.pack([tag, epochtime, data.merge(LogStash::Event::TIMESTAMP => event.timestamp.to_iso8601)]))
   end # def encode
 
+  def forwardable_tag(event)
+    tag = event.get("tags") || "log"
+    case tag.class
```

Comment:
when using a case statement to test for an object class, `.class` should not be used:
```
case tag
when Array
  ...
when String
 ...
end
```

Reply:
Oops, I’ll fix it.

Reply:
I've fixed it.